easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-6.4.0-2.28-LAPACK-3.7.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenBLAS'
+version = '0.2.19'
+
+lapackver = '3.7.0'
```

Comment:
Or not, we should probably stick with the shipped LAPACK instead, see https://github.com/xianyi/OpenBLAS/issues/1273...